mods/player_api/api.lua

```diff
@@ -104,6 +110,12 @@ function player_api.set_textures(player, textures)
 	player:set_properties({textures = new_textures})
 end
 
+function player_api.set_texture(player, index, texture)
```

Comment:
I'd invert `index` and `texture`, to make it coherent with `set_textures`

Reply:
Imagining this as a table, I consider `playertextures[index] = texture` more intuitive.